scripts/layouts_tool/defs.h

```diff
@@ -0,0 +1,65 @@
+#ifndef DEFS_H
+#define DEFS_H
+
+static const int LAYOUT_LEN = 48;
```

Comment:
In Mixxx we are replaceing the static keyword to hide things from the linker by anonymous namespaces.
Capital Texts are macros Constants have a k prefix.  
```
namespace {
    const int kLayoutLen = 48;
}